PWGHF/vertexingHF/vHFML/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ include_directories(${ROOT_INCLUDE_DIRS}
                     ${AliPhysics_SOURCE_DIR}/OADB
                     ${AliPhysics_SOURCE_DIR}/PWGHF/vertexingHF
                     ${AliPhysics_SOURCE_DIR}/PWGHF/vertexingHF/charmFlow
+                    ${AliPhysics_SOURCE_DIR}/PWGHF/vertexingHF/vHFBDT
```

Comment:
You should include the task in vertexingHF and not vHFML for the moment, since you don't need what is in vHFML and we have the issue with treelite still unsolved.